<?xml version="1.0" ?>

<launch>
  <arg name="publish_rate" default="100" />
  <arg name="mpc_yml_file" default="$(find franka_motion_control)/config/franka_reacher.yml" />
  

  <node name="mpc" type="mpc.py" pkg="franka_motion_control" output="screen">
    <rosparam command="load" file="$(find franka_motion_control)/config/robot_settings.yaml" />
    <param name="mpc_yml_file" value="$(arg mpc_yml_file)" />
  </node>


</launch>
